airflow/cli/commands/dag_command.py

```diff
@@ -272,7 +272,7 @@ def dag_state(args, session: Session = NEW_SESSION) -> None:
     >>> airflow dags state tutorial 2015-01-01T00:00:00.000000
     running
     >>> airflow dags state a_dag_with_conf_passed 2015-01-01T00:00:00.000000
-    failed, {"name": "bob", "age": "42"}
+    failed, {"name": "bob", "age": "42"}.
```

Comment:
This shouldn't end with a `.` as it is code.

Reply:
Good catch! I fixed it. I also went through all the changes and double checked them. It looks good